test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -624,7 +624,7 @@ int test_main(int, char* [])
 #if defined(BOOST_GEOMETRY_TEST_FAILURES)
     // Not yet fully tested for float and long double.
     // The difference algorithm can generate (additional) slivers
-    BoostGeometryWriteExpectedFailures(10, 11, 24, 15);
+    BoostGeometryWriteExpectedFailures(10, 11, 24, 16);
```

Comment:
I still have to check if this was a regression (and then: why) or something else.

Reply:
It fails in another (better) way and therefore the #expectation differs (one test can have multiple failures, for area, number of output polygons, etc)